runtime/jailer.go

```diff
@@ -81,6 +83,10 @@ func newJailer(
 		return newNoopJailer(ctx, l, service.shimDir), nil
 	}
 
+	if err := os.MkdirAll(ociBundlePath, 0644); err != nil {
+		return nil, errors.Wrap(err, "failed to create oci bundle path")
```

Comment:
How about including the path itself in the error, as like errors in runc_jailer.go?